### PR TITLE
Make urls protocol relative

### LINKED ISSUE #231: Hardcoded non-ssl urls
There are a few different places where `http` is hardcoded:

Can be made protocol relative:
https://github.com/prebid/Prebid.js/blob/b8de2b6c286a390e0cb19d0a69a76580fd38ad13/src/adapters/pulsepoint.js#L7-L8
https://github.com/prebid/Prebid.js/blob/b8de2b6c286a390e0cb19d0a69a76580fd38ad13/src/adapters/springserve.js#L10

Unsure, can't find vendor docs:
https://github.com/prebid/Prebid.js/blob/b8de2b6c286a390e0cb19d0a69a76580fd38ad13/src/adapters/rubiconLegacy.js#L96

If anyone has a clue about the last one I'd be glad to fix it.

### PR DESCRIPTION
**Pull request by dwick**
fixes #231

### CODE REVIEW

File: src/adapters/springserve.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ SpringServeAdapter = function SpringServeAdapter() {
 
   function buildSpringServeCall(bid) {
 
-    var spCall = 'http://bidder.springserve.com/display/hbid?';
+    var spCall = '//bidder.springserve.com/display/hbid?';
```
Comment (mkendall07) [inline src/adapters/springserve.js]: please hardcode to `https` or use `document.protocol`

File: src/adapters/pulsepoint.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ var adloader = require('../adloader.js');
 
 var PulsePointAdapter = function PulsePointAdapter() {
 
-  var getJsStaticUrl = 'http://tag.contextweb.com/getjs.static.js';
-  var bidUrl = 'http://tag.contextweb.com/bid';
+  var getJsStaticUrl = '//tag.contextweb.com/getjs.static.js';
```
Comment (mkendall07) [inline src/adapters/pulsepoint.js]: these changes have already been made via https://github.com/prebid/Prebid.js/pull/230

**Comment by mkendall07** [comment]
@dwick 
Thanks for the PR. Please update to resolve conflict and address comment. Thanks

**Comment by dwick** [comment]
@mkendall07 done

**Pull request merged**